extensions/amp-ad/0.1/amp-ad.css

```diff
@@ -48,8 +56,18 @@ amp-embed iframe {
   padding: 2px 4px;
 }
 
-amp-ad[data-a4a-upgrade-type='amp-ad-network-adsense-impl'] > iframe,
-amp-ad[data-a4a-upgrade-type='amp-ad-network-doubleclick-impl'] > iframe {
+.i-amphtml-ad-default-holder[experiment1]:after,
+.i-amphtml-ad-default-holder[experiment2]:after {
+  font-size: 10px;
+  line-height: 10px;
+  padding: 3px 4px 0px 4px;
```

Comment:
@spacedino FYI: I changed the padding-left from 3px to 4px. I personally felt different value makes the text not centered. Please let me know if you still think `padding-left: 3px` is better.
`padding-left: 3px`
![image](https://cloud.githubusercontent.com/assets/5419904/24779007/7a8f8846-1ae2-11e7-9c65-d4fd145f906f.png)
`padding-left: 4px`
![image](https://cloud.githubusercontent.com/assets/5419904/24779021/84658dc0-1ae2-11e7-85dc-c6f6b5c38085.png)